pkgs/test/src/updateChecker.ts

```diff
@@ -0,0 +1,19 @@
+import { Octokit } from "@octokit/rest"
+
+const latestRelease = async () => {
+    const gitHub = new Octokit().rest
+    const { data } = await gitHub.repos.getLatestRelease({
+        owner: "re-do",
+        repo: "redo"
+    })
+    return data
+}
+
+export const promptIfNewReleaseAvailable = async (version: string) => {
```

Comment:
I know this is kind of troll at this point but I think the package we should be checking to determine whether a new release is available is "@re-do/test," not the GitHub releases of the app. That way, if we publish a new release of the app but don't have the corresponding version of test published yet, we won't confuse people by giving them this prompt over and over (in that case, "npm i @re-do/test@latest" would do nothing).

Luckily, I think it should be really straightforward to check if there is a newer @re-do/test available just using the npm cli via the shell.